changelogs/unreleased/5470-remove-featureflag.yml

```diff
@@ -0,0 +1,6 @@
+description: Remove Instance Composer feature flag and make it accessible to all users
```

Comment:
I wouldn't say all users because this isn't available on OSS for example. 
Perhaps saying the Composer is not in beta version anymore might be better?

Reply:
Right, forgot about oss. I'll change that, it's bold, but I guess we are out of beta if we are making it accessible by default

Reply:
@bartv do you have any opinion on that?

Reply:
I would write: Smart Composer is now general available.

Reply:
Changed